questions/includes/fedora/coding/python.yml

```diff
@@ -14,7 +14,7 @@ tree:
 
     - title: Firewalld
       subtitle: a dynamically managed firewall with support for network zones
-      href: http://www.firewalld.org/contribute/
```

Comment:
oh wait, pbkac...

with javascript turned on it does load the page correctly. sorry for the inconvenience. =(